pkg/registry/core/next/discovery_server.go

```diff
@@ -40,7 +40,9 @@ type nextDiscoveryServer struct {
 func NewWrappedDiscoveryServer(wrapper DiscoveryServerWrapper, servers ...registry.NetworkServiceDiscoveryServer) registry.NetworkServiceDiscoveryServer {
 	rv := &nextDiscoveryServer{servers: make([]registry.NetworkServiceDiscoveryServer, 0, len(servers))}
 	for _, c := range servers {
-		rv.servers = append(rv.servers, wrapper(c))
+		if c != nil {
```

Comment:
Do we ignore with this check that SDK developer doing something wrong?

Reply:
There was no such check, but it simplify usage of this kind of wrapper servers a lot.